frontend/lib/verification/verification_qr_code_processor.dart

```diff
@@ -1,40 +0,0 @@
-import 'package:ehrenamtskarte/proto/card.pb.dart';
```

Comment:
This file was moved to `identification/verification_workflow/verification_qr_code_processor.dart`